### DISCUSSION TITLE
iphone alternatives

### DISCUSSION THREAD

**Discussion by eadmaster** [Waboodoo/HTTP-Shortcuts#268] [Q&A]
do you know any similar app for the iphone that can be used as a share target?

**Comment by Waboodoo** [comment]
I have never looked into this. In fact, I'm not particularly familiar with the Apple ecosystem in general.

**Comment by radi5** [comment]
Hi @eadmaster,
I had the same problem. Having no knowledge about apple ios at all but the need to automate something. I used 
https://support.apple.com/en-gb/guide/shortcuts/welcome/ios
Maybe that helps.
BR
Radi
